Sources/PostgresNIO/New/Messages/NotificationResponse.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 26.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```